js/spa/test/test_chatinterface_streaming_echo.spec.ts

```diff
@@ -104,6 +104,7 @@ test("test stopping generation", async ({ page }) => {
 	await stop_button.click();
 
 	await expect(page.locator(".bot.message").first()).toContainText("abc");
+	await page.waitForTimeout(1000);
```

Comment:
I feel like this could be a shorter timeout? 
```suggestion
	await page.waitForTimeout(1000);
```